api/src/backend/api/v1/views.py

```diff
@@ -2565,3 +2572,126 @@ def get_serializer_context(self):
         context = super().get_serializer_context()
         context["allowed_providers"] = self.allowed_providers
         return context
+
+
+class LighthouseConfigViewSet(BaseRLSViewSet):
+    """
+    API endpoint for managing OpenAI API configuration.
+    """
+
+    filterset_fields = {
+        "name": ["exact", "icontains"],
+        "model": ["exact", "icontains"],
+        "is_active": ["exact"],
+        "inserted_at": ["gte", "lte"],
+    }
+    ordering_fields = ["name", "inserted_at", "updated_at", "is_active"]
+    ordering = ["-inserted_at"]
+
+    def get_queryset(self):
+        return LighthouseConfig.objects.all()
```

Comment:
```suggestion
        return LighthouseConfig.objects.filter(tenant_id=self.request.tenant_id)
```

Every tenant would have its own config, isn't it? This would improve performance.